doc/en/mark.rst

```diff
@@ -42,8 +40,10 @@ Marks can be registered in ``pytest.ini`` like this:
         slow
         serial
 
-This can be used to prevent users mistyping mark names by accident. Test suites that want to enforce this
-should add ``--strict`` to ``addopts``:
+When the ``--strict`` command-line flag is passed, any unknown marks applied
```

Comment:
@Zac-HD how about using this PR and add `--strict-marks` as an alias to `--strict`, while also updating the docs to use the former?

Reply:
I'd prefer to leave functional changes to the other PR - will discuss there.

Reply:
Sure, thanks!